src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PlacementApplicationEntity.kt

```diff
@@ -126,6 +126,10 @@ data class PlacementApplicationEntity(
   @OneToMany(mappedBy = "placementApplication")
   var placementDates: MutableList<PlacementDateEntity>,
 
+  /**
+   * Supporting multiple placements requests in a single Placement Application is legacy behaviour. Any new
+   * placement application will only ever one and only one date in this collection
```

Comment:
did you mean to say something along the lines of "will only have one date in this"?

Reply:
Good spot, fixed